app/bebbo/dynamicImports.ts

```diff
@@ -18,3 +27,31 @@ export const standardDevData =require('@offlinedata/bebbo/standardDevData').stan
 export const SurveyData =require('@offlinedata/bebbo/SurveyData').SurveyData;
 export const vaccineData =require('@offlinedata/bebbo/vaccineData').vaccineData;
 export const VideoArticleData =require('@offlinedata/bebbo/VideoArticleData').VideoArticleData;
+import northMacedinaPartnerLogo from '@assets/appOfflineImages/northmacedonia/partner/1.png';
+export const nmPartnerLogo=northMacedinaPartnerLogo;
+import northMacedinaSponserLogo from '@assets/appOfflineImages/northmacedonia/sponser/2.png';
+export const nmSponserLogo=northMacedinaSponserLogo;
+import albeniaPartnerLogo from '@assets/appOfflineImages/albenia/partner/IPH.png';
```

Comment:
Let's rename fodler albenia to albania
and 
albeniaPartnerLogo to albaniaPartnerLogo